### PR TITLE
Update CHANGELOG.md (1.6.0-beta02)

### PR DESCRIPTION
**Pull request by igordmn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,75 @@
+# 1.6.0-beta02 (February 2024)
+
+## Highlights
+- [Basic accessibility support](https://github.com/JetBrains/compose-multiplatform-core/pull/1025) <sub>_iOS_</sub>
+- [Popups/Dialogs can now be displayed outside the main window or controller over native components on iOS by default](https://github.com/JetBrains/compose-multiplatform-core/pull/1031) <sub>_iOS_</sub>
+- [Allow selecting text in Selection Container by double and triple tap](https://github.com/JetBrains/compose-multiplatform-core/pull/984) <sub>_iOS_</sub>
```
Comment (pjBooms) [inline CHANGELOG.md:6]: Bug fixes in the resource library

Comment (igordmn) [reply CHANGELOG.md:6]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+## HTML library
+### Features
+- [Add opportunity to use custom prefixes in `StyleSheet`](https://github.com/JetBrains/compose-multiplatform/pull/3015)
+
+## Gradle Plugin
+
+### Features
+- [Add `ui-tooling-preview` alias](https://github.com/JetBrains/compose-multiplatform/pull/4190)
+
+## Resource library
```
Comment (pjBooms) [inline CHANGELOG.md:51]: We need to mention that .ttf fonts work (with a link to https://github.com/JetBrains/compose-multiplatform/commit/3c7260ea51157d423b3799bd339b682ffabdce06)

Comment (pjBooms) [reply CHANGELOG.md:51]: Added a ttf font to the resources demo app

Comment (igordmn) [reply CHANGELOG.md:51]: Done

**Review by pjBooms — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+- _(prerelease bug)_ [Fix selection handles crossed](https://github.com/JetBrains/compose-multiplatform-core/pull/1017)
+- _(prerelease bug)_ [Fix CMPViewControllerMisuse error](https://github.com/JetBrains/compose-multiplatform-core/pull/1027)
+- _(prerelease bug)_ [Fix selection handles with platformLayers=true](https://github.com/JetBrains/compose-multiplatform-core/pull/1023)
+- _(prerelease bug)_ [Fix interaction handling for interop views](https://github.com/JetBrains/compose-multiplatform-core/pull/1032)
+
+## Desktop
+
+### Fixes
+- [Add Arial and Consolas as backup fonts on Linux and mention font name when one fails to load](https://github.com/JetBrains/compose-multiplatform-core/pull/994)
+
```
Comment (MatkovIvan) [inline CHANGELOG.md:38]:
```suggestion
- _(prerelease bug)_ [Fix initialization of Direct3D swap chain before getting the canvas size](https://github.com/JetBrains/skiko/pull/858)

```

Comment (igordmn) [reply CHANGELOG.md:38]: The new Skiko wasn't merged for beta02

Comment (MatkovIvan) [reply CHANGELOG.md:38]: Ah, right..

**Pull request merged**